test/src/utils.ts

```diff
@@ -177,3 +179,34 @@ export function getUniqueVideoFilePlaceholder(): FilePlaceholder {
 export function rmIfExists(file: string): Promise<void> {
   return rm(file).catch(() => {});
 }
+
+export function waitForFileExistence(
+  filePath: string,
+  timeout = 5000,
+): Promise<boolean> {
+  return new Promise((resolve, reject) => {
+    access(filePath, constants.R_OK, err => {
```

Comment:
let's use the promises API https://nodejs.org/docs/latest/api/fs.html#promises-api for both access and watch.

Reply:
Done